Tests/bootstrap.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Bootstrap file for running the tests.
+ *
+ * Load the PHPCS autoloader and the WPCS PHPCS cross-version helpers.
```

Comment:
You asked about giving credit, but don't mention that the bulk of the code in this file comes from/is inspired by WPCS at all now....